perllib/FixMyStreet/SendReport/Email.pm

```diff
@@ -110,6 +110,7 @@ sub send {
         $params, $sender, $nomail, $cobrand, $row->lang);
 
     unless ($result) {
+        $row->set_extra_metadata('sent_to' => email_list($params->{To}));
```

Comment:
This will be overwritten if we re-send the message. Not sure how much of a concern that is. Given it's in theory for tracking who is currently responsible for it, possibly not a concern.

Reply:
Yeah, I thought it best for now to match what happens to e.g. `whensent` and `send_method_used` which both only contain the lastest sending information.